src/development/tools/devtools/console.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Using the Debug console
+description: Learn how to use the DevTools console to xxx.
+---
+
+The DevTools Debug console allows you to watch an
+application's standard output (`stdout`),
+evaluate expressions for a paused or running
+app in debug mode, and analyze inbound and outbound
+references for objects.
+
+{{site.alert.note}}
+  This page is up to date for DevTools 2.23.0.
+{{site.alert.end}}
+
+The Debug console is available from the [Inspector][],
+[Debugger][], and [Memory][] views. To bring up
+the debug console, click the **More info** link,
+as shown:
+
+![Screenshot of how to bring up Debug console]({{site.url}}/assets/images/docs/tools/devtools/debug-console.png){:width="100%"}
```

Comment:
Please, remove this text and picture. This was explanation of the context for the tech writers on how user will access the documentation:

"To bring up
the debug console, click the **More info** link,
as shown:"